rfcs/syntax-default-type-alias-type-parameters.md

```diff
@@ -0,0 +1,78 @@
+# Default type alias type parameters
```

Comment:
We'll probably want to apply the default type arguments to generic functions too. This title implies restricting this feature to type aliases only. I assume that's unintentional.

```suggestion
# Default type parameters
```

Also rename the file as well.